recipes/proj/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "9.3.1":
+    url: "https://github.com/OSGeo/PROJ/archive/9.3.0.tar.gz"
+    sha256: "f48c334eaf56c38d681bcfa37f188f422a562f45a66e6e646a79b4249641ecdb"
```

Comment:
Labeled 9.3.1, but this is just 9.3.0 copy and pasted. should be 
```yaml
  "9.3.1":
    url: "https://github.com/OSGeo/PROJ/releases/download/9.3.1/proj-9.3.1.tar.gz"
    sha256: "b0f919cb9e1f42f803a3e616c2b63a78e4d81ecfaed80978d570d3a5e29d10bc"
```
The cmake targets patch from 9.2.1 still applies luckily.

I was going to make a PR just for 9.3.1 because it has some bug fixes that we would like to use. Should we just wait for this PR to get in?

Reply:
Ugh. Thanks!
Yes, feel free to create a separate PR for the version bump. It might be possible to auto-merge it if it does not require any new patches.